manager/volume.go

```diff
@@ -742,6 +746,120 @@ func (m *VolumeManager) EngineUpgrade(volumeName, image string) (v *longhorn.Vol
 	return v, nil
 }
 
+func (m *VolumeManager) checkVolumeNotInMigration(volumeName string) error {
```

Comment:
I feel like it belongs in the volume manager since it's dealing with the volume state.
If you feel strongly about this, let me know and I will move it :)

Reply:
done